manifests/canary-checker.yaml

```diff
@@ -228,13 +242,18 @@ spec:
   jobLabel: canary-checker
   selector:
     matchLabels:
+      app.kubernetes.io/instance: canary-checker
+      app.kubernetes.io/name: canary-checker
       control-plane: canary-checker
 ---
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: nginx
```

Comment:
Is this patched in?  It should not be the default in the base template...and I don't see it in the values.yaml

Reply:
It's the default in the canary-checker helm chart values. It probably shouldn't be default there and be left to downstream consumers to set

Reply:
https://github.com/flanksource/canary-checker/pull/443

Reply:
This has now been dropped upstream and updated here